GameServerLib/Items/InventoryManager.cs

```diff
@@ -29,16 +32,31 @@ public IItem GetItem(byte slot)
             return _inventory.GetItem(slot);
         }
 
+        public IItem GetItem(string itemSpellName)
```

Comment:
Replace body of function with `return _inventory.GetItem(itemSpellName);`

Reply:
resolved

Reply:
Not actually resolved.

Reply:
Oops my bad